code/common/datadog.q

```diff
@@ -29,6 +39,20 @@ isok:{$[.proc.proctype in key .dg.handlers;.dg.handlers .proc.proctype;1b]}
     .lg.w[`sendmetric;"Currently only linux operating systems are supported to send metrics"]]
   }
 
+//define sendmetric and sendevent functions using web request
+.dg.sendevent_webreq:{[event_title;event_text;tags;alert_type]
+  url:`$(dogstatsd_url,"events?api_key=",dogstatsd_apikey);
+  .Q.hp[url;.h.ty`json]
+  .j.j (`title;`text;`priority;`tags;`alert_type)!(event_title;event_text;"normal";$[0h=type tags;","sv tags;tags];alert_type)
```

Comment:
Few clean up things here:
1. indent second line to make it clear it follows on from the first
2. list of symbols doesn't need to be wrapped in `()`
```suggestion
  .Q.hp[url;.h.ty`json]
    .j.j`title`text`priority`tags`alert_type!(event_title;event_text;"normal";$[0h=type tags;","sv tags;tags];alert_type)
```